Client/Frontend/Fakespot/Views/NoAnalysisCardView.swift

```diff
@@ -0,0 +1,153 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Common
+import Shared
+import ComponentLibrary
+
+class NoAnalysisCardViewModel {
```

Comment:
Can't this be a struct? So we get the init for free. 
Also, this can be in a separate file.